test/api/v2/groups/POST-groups.test.js

```diff
@@ -24,61 +21,58 @@ describe('POST /groups', () => {
       });
     });
 
-    it('returns a group object', () => {
-      let group = {
+    it('returns a group object', async () => {
+      let group = await leader.post('/groups', {
         name: 'Test Group',
         type: 'party',
         leaderOnly: { challenges: true },
         description: 'Test Group Description',
         leaderMessage: 'Test Group Message',
-      };
-
-      return leader.post('/groups', group).then((createdGroup) => {
-        expect(createdGroup._id).to.exist;
-        expect(createdGroup.leader).to.eql(leader._id);
-        expect(createdGroup.name).to.eql(group.name);
-        expect(createdGroup.description).to.eql(group.description);
-        expect(createdGroup.leaderMessage).to.eql(group.leaderMessage);
-        expect(createdGroup.leaderOnly).to.eql(group.leaderOnly);
-        expect(createdGroup.memberCount).to.eql(1);
       });
+
+      expect(group._id).to.exist;
+      expect(group.leader).to.eql(leader._id);
+      expect(group.name).to.eql(group.name);
+      expect(group.description).to.eql(group.description);
+      expect(group.leaderMessage).to.eql(group.leaderMessage);
+      expect(group.leaderOnly).to.eql(group.leaderOnly);
+      expect(group.memberCount).to.eql(1);
     });
 
-    it('returns a populated members array', () => {
-      return leader.post('/groups', {
+    it('returns a populated members array', async () => {
+      let party = await leader.post('/groups', {
```

Comment:
Personally, I like this better.

For the errors, I thought you needed to do the `return <Promise>` syntax, but since it's a promise, you can just use `await` for those as well. I'll update the PR. Thanks for pointing it out.